### PR TITLE
feature/BOT-3106-core-box-add-a-wer-matching-mod

### PR DESCRIPTION
**Pull request by murliwatz**

### CODE REVIEW

File: src/scripting/logichook/asserter/WerAsserter.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      return Promise.reject(new BotiumError(`${convoStep.stepTag}: WerAsserter Missing argument`,
+        {
+          type: 'asserter',
+          subtype: 'wrong parameters',
+          source: this.name,
+          cause: { args }
+        }
+      ))
+    }
+    if (args.length > 2) {
```
Comment (ujja) [inline src/scripting/logichook/asserter/WerAsserter.js:23]: 0 is invalid as I see, check it too?

File: src/scripting/logichook/asserter/WerAsserter.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      ))
+    }
+
+    const utterance = args[0]
+    const threshold = args[1]
+
+    const wer = speechScorer.wordErrorRate(botMsg.messageText, utterance)
+    if (wer > threshold) {
+      return Promise.reject(new BotiumError(
+        `${convoStep.stepTag}: Word error rate ${wer} > ${threshold} for ${utterance}`,
```
Comment (ujja) [inline src/scripting/logichook/asserter/WerAsserter.js:40]: ">" is "=>"?

File: src/scripting/logichook/asserter/WerAsserter.js (hunk 1 above)
Comment (murliwatz) [reply src/scripting/logichook/asserter/WerAsserter.js:23]: > arg[0] is invalid as I see, check it too?

It is checked above

File: src/scripting/logichook/asserter/WerAsserter.js (hunk 2 above)
Comment (murliwatz) [reply src/scripting/logichook/asserter/WerAsserter.js:40]: I will check the ranges again

**Pull request merged**